translator/totomlconfig/tomlConfigTemplate/tomlConfig.go

```diff
@@ -323,12 +323,13 @@ type (
 	}
 
 	k8sDecoratorConfig struct {
-		ClusterName       string `toml:"cluster_name"`
-		HostIp            string `toml:"host_ip"`
-		NodeName          string `toml:"host_name_from_env"`
-		Order             int
-		PreferFullPodName bool `toml:"prefer_full_pod_name"`
-		TagService        bool `toml:"tag_service"`
-		TagPass           map[string][]string
+		ClusterName             string `toml:"cluster_name"`
+		DisableMetricExtraction bool   `toml:"disable_metric_extraction"`
```

Comment:
Would be better to add in `schema.json` (as the rest of others options) since its the source of truth for customers to configure their CWAgent json beside the public document

Reply:
Will check with our PM and if that is indeed the case, will batch that update along with other options and also once we add this for ECS & Prometheus.